### ISSUE TITLE
Top-level breadcrumb collection colors could cause confusion

### ISSUE THREAD

**Issue by ali-h2010** [BookStackApp/BookStack#1435]
**Describe the feature you'd like**
1- Add an indication to which page I am currently in inside the Page Navigation.
2- A link to the next page or chapter at the bottom of each page will make the navigation easier.  
3- Main page color(green) and icon are very similar to Books color and icon which is confusing. 
Screenshots attached for clarification

**Describe the benefits this feature would bring to BookStack users**
It will make the view clearer, navigation faster, and improve user experience overall.

**Additional context**
![Screen Shot 2019-05-08 at 10 53 11 AM](https://user-images.githubusercontent.com/12033261/57359690-fd3e9380-7180-11e9-8f74-21fe35d6c223.png)

![Screen Shot 2019-05-08 at 11 06 14 AM](https://user-images.githubusercontent.com/12033261/57359944-a4232f80-7181-11e9-922c-abbcb2eeea6f.png)

**Comment by ssddanbrown** [comment]
Hello @ali-h2010,

Thank you for your recommendations. 

> 1- Add an indication to which page I am currently in inside the Page Navigation.

Absolutely! This is something I accidentally removed during the recent redesign. I have just applied a fix to the master branch so this will be in the next release. 

> 2- A link to the next page or chapter at the bottom of each page will make the navigation easier.

Yeah, This is a good idea and has already been requested in issue #1381.

> 3- Main page color(green) and icon are very similar to Books color and icon which is confusing.
Screenshots attached for clarification

I see how this could be confusing. The green is the color of a books so I've applied the same to a collection of books. Not really sure what the best alternative to use here. This will be the same situation for shelves. 

Since items 1 & 2 are now already covered I'll update the title of this issue to focus on item 3.
